src/components/TaskModule/components/TaskItem/index.tsx

```diff
@@ -161,15 +161,31 @@ const TaskItem = ({
   );
 
   const jumpAndRefresh = async (taskId: USER_TASK_DETAIL) => {
-    const jumpItem = jumpExternalList.find(
-      (item) => item.taskId === data.userTaskDetail
-    );
-    if (jumpItem) {
-      openNewPageWaitPageVisible(
-        jumpItem.url,
-        taskId,
-        () => sendCompleteReq(taskId)
+    try {
+      const jumpItem = jumpExternalList.find(
+        (item) => item.taskId === data.userTaskDetail
       );
+      if (jumpItem) {
+        const isComplete = await openNewPageWaitPageVisible(
+          jumpItem.url,
+          taskId,
+          () =>
+            fetchWithToken(
+              `/api/app/user/complete-task?${new URLSearchParams({
+                chainId,
+                userTask: userTask,
```

Comment:
if it is the same name you do not need to pass in like this, cna just userTask

Reply:
Updated